### PR TITLE
Update composer version, link to packagist

### PR DESCRIPTION
**Pull request by aretecode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +21,9 @@ Read the blog post about this repository at:
 ## Installation
 
 ```
-$ composer require broadway/broadway ~0.2.0
+$ composer require broadway/broadway ~0.5.2
```
Comment (wjzijderveld) [inline README.md]: Could you change it to just `composer require broadway/broadway`? Composer uses the latest version by default then. Saves us updating this regularly :)

**Comment by wjzijderveld** [comment]
Thanks for the PR. If you could make that change, I will do my best to get Travis actually succeed a build :)

File: README.md [hunk 2]
```diff
@@ -21,8 +21,9 @@ Read the blog post about this repository at:
 ## Installation
 
 ```
-$ composer require broadway/broadway ~0.2.0
+$ composer require broadway/broadway
 ```
+[packagist]: https://packagist.org/packages/broadway/broadway
```
Comment (wjzijderveld) [inline README.md]: Sorry, I missed this when I reviewed it earlier, but you define a link which isn't used anywhere...

**Comment by wjzijderveld** [comment]
@aretecode Could you provide an update for that latest comment?

**Comment by aretecode** [comment]
@wjzijderveld since it does not show the version (previously it showed an outdated version) this shows the current version and previous versions. It is optional, the most important thing was fixing the version.

**Comment by wjzijderveld** [comment]
@aretecode I'm not sure if you understood my comment.
What I meant, is that you define a link in Markdown (to packagist), but that link isn't used anywhere.

So please, use the link to actually have a link in the README to packagist, or remove the link.

**Pull request closed without merging**